substrate/frame/proxy/src/tests.rs

```diff
@@ -26,10 +26,14 @@ use alloc::{vec, vec::Vec};
 use codec::{Decode, Encode};
 use frame_support::{
 	assert_noop, assert_ok, derive_impl,
-	traits::{ConstU32, ConstU64, Contains},
+	traits::{fungible::HoldConsideration, ConstU32, Contains, Currency},
 };
 use sp_core::H256;
-use sp_runtime::{traits::BlakeTwo256, BuildStorage, DispatchError, RuntimeDebug};
+use sp_runtime::{
```

Comment:
please make sure that we have a test coverage for the things we altered. 
we can introduce some general mock consideration impl. on a pallet level we do not care about right deposit balance calculation, etc, but tests should check that the ticket was updated and the updated record was persisted to the storage. or a pallet care that a drop function was called and a ticket was removed from the storage.